ddp-workspace/projects/ddp-dsm-ui/src/app/shipping-search/shipping-search.component.html

```diff
@@ -144,10 +144,13 @@ <h3 class="Color--primary Line--Break">{{additionalMessage}}</h3>
                   (click)="receiveATKit(kitRequest)">Mark Received</button>
         </td>
         <td>
-          <app-field-datepicker (dateChanged)="valueChanged($event, 'collectionDate', kitRequest)"
+          <app-field-datepicker (input)="changingDate()"
```

Comment:
Thank you for pointing out the bug and better format for the (input) trigger. I fixed the message showing up for each row bug by adding a "saved" field to the specific kitRequest object the corresponds with the row that is currently being edited. Then I just check that specific object for the "saved" flag rather than a global one within the shipping-search.component. I assume this is okay since we do not write the entire kit request back to the server, just field updates.